VMCreate.yml

```diff
@@ -0,0 +1,61 @@
+---
+- name: Create Azure VM
+  hosts: localhost
+  connection: local
+  vars:
+    resource_group: demo
```

Comment:
I think we could add comment here:
# change variables below to customize your VM deployment
or something like this

Reply:
haha, i have added # at the beginning of the line, that's why so big ;-)

Reply:
love the size of font!

Reply:
Thank you for suggestion. Add comment for 2 playbook files.